.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
Yeah, if You prefer not to keep submodules, it could be installed through npm before test execution since both bats and bats-mock are available as npm modules.